cli/tags.py

```diff
@@ -1,9 +1,44 @@
 import click
+from ._utils import add_options
+from rich.console import Console
+from rich.table import Table
+from rich import box
 
+__tags_options = [
+    click.option(
+        "-a",
+        "--all",
+        is_flag=True,
+        help="""
+    List all tags
+    """,
+    ),
+]
 
-@click.group("tags", invoke_without_command=True)
-def tags():
-    """
-    List tags
-    """
-    pass
+
+@click.command(short_help="Get tags")
+@add_options(__tags_options)
+@click.pass_context
+@click.argument("value", required = False)
+def tags(ctx, value, all):
+    if all:
+        error, answer = ctx.obj.get_all_tags()
+
+    else:
+        if value:
+            error, answer = ctx.obj.get_tag_by_id(value)
+    print_table(answer)
+
+
+
+def print_table(data):
+    console = Console()
+    table = Table(show_header=True)
+    table.add_column("Id", no_wrap=True, header_style = "bold blue")
+    table.add_column("Label", no_wrap=True, header_style = "bold blue")
+    table.add_column("Color", no_wrap=True, header_style = "bold blue")
+    for elem in data:
+        table.add_row(str(elem['id']), str(elem['label']), str(elem['color']))
```

Comment:
I want a feature here, say: `elem['color']` is `red` then that row's font text color should be `red`. This should be easily doable since `rich` a way to add color to text easily.